village/src/App.js

```diff
@@ -3,22 +3,38 @@ import React, { Component } from 'react';
 import './App.css';
 import SmurfForm from './components/SmurfForm';
 import Smurfs from './components/Smurfs';
+import axios from "axios";
+import { Route } from "react-router-dom";
+import NavBar from "./components/NavBar";
 
 class App extends Component {
   constructor(props) {
     super(props);
     this.state = {
-      smurfs: [],
+      smurfs: []
     };
   }
+
+  componentDidMount() {
+    this.getData("http://localhost:3333/smurfs");
+  }
+
+  getData = URL => {
+    axios
+      .get(URL)
+      .then(response => this.setState({ smurfs: response.data }))
+      .catch(err => console.log(err));
+  };
+
   // add any needed code to ensure that the smurfs collection exists on state and it has data coming from the server
   // Notice what your map function is looping over and returning inside of Smurfs.
   // You'll need to make sure you have the right properties on state and pass them down to props.
   render() {
     return (
       <div className="App">
-        <SmurfForm />
-        <Smurfs smurfs={this.state.smurfs} />
+      <Route path="/" component={NavBar} />
```

Comment:
You can have the NavBar component outside a route and have it render everywhere also, but if works this way as well because you did not put the exact parameter on it, very inventive, nice job